### PR TITLE
Safari Back Button Fix

### LINKED ISSUE #289: Mac Safari - Cannot get back to results from article "full page"
Steps: (in safari browser **Version 9.1.1** [latest])
1. visit: http://www.tcdl.io.s3-website-eu-west-1.amazonaws.com/isearch/0.17/index.html#/article/9139c770-32d3-11e6-9c16-19d96769d918/tile:article.dk.121
2. Attempt to click back button to go back to the results...
mac-safari-cannot-use-back-button-in-article-view

> Note: this worked for me in Google Chrome on Mac.

### PR DESCRIPTION
**Pull request by jackcarlisle**
## ready for review
fixes #289 

* changes from goBack() to go(-2) to support Safari and IOS

### CODE REVIEW

File: lib/nav-header/index.js [hunk 1]
```diff
@@ -5,11 +5,12 @@ require('./style.css');
 
 export default class NavHeader extends Component {
   render () {
-    const { backToSearch } = this.props;
+    const { go } = this.props;
     return (
       <div className='navBarLayout'>
         <div className='navBarContainer'>
-          <Link to='/' className='backButton' onClick={() => backToSearch()}>
+        // using go(-2) instead of goBack() because it is not supported by safari or IOS
```
Comment (nelsonic) [inline lib/nav-header/index.js]: please link to the actual issue in the backlog too.

**Comment by nelsonic** [comment]
@jackcarlisle code looks good. (_well done for figuring it out_)
Please link to the issue in the _in-line_ comment e.g: https://git.io/voEOD so it's easy to see why that code is the way it is... 👍

**Comment by nelsonic** [comment]
@jackcarlisle please resolve merge conflicts and ping me.

**Comment by jackcarlisle** [comment]
@nelsonic resolved!

**Comment by nelsonic** [comment]
@jackcarlisle thanks! 👍

**Pull request merged**